onnx/defs/tensor/defs.cc

```diff
@@ -34,33 +34,74 @@ User must be aware of precision loss and value change caused by range difference
 For example, a 64-bit float 3.1415926459 may be round to a 32-bit float 3.141592. Similarly, converting
 an integer 36 to Boolean may produce 1 because we truncate bits which can't be stored in the targeted type.
 
-In more detail, the conversion among numerical types should follow these rules:
+In more detail, the conversion among numerical types should follow these rules
+if the destination type is not a float 8 type.
 
 * Casting from floating point to:
-  * floating point: +/- infinity if OOR (out of range).
+  * floating point: +/- highest value if OOR (out of range),
+    infinities are converted to NaN if infinity is not available
   * fixed point: undefined if OOR.
   * bool: +/- 0.0 to False; all else to True.
 * Casting from fixed point to:
-  * floating point: +/- infinity if OOR. (+ infinity in the case of uint)
+  * floating point: +/- highest value if OOR (out of range),
```

Comment:
Now that we have separated the fp8 rules into a separate paragraph down below, is there any reason to modify this section? Is infinity not available in any other data type? Also I think "+/- infinity" would be clearer than "+/- highest value" since "highest value" sounds a bit like max norm.

nit: I think we can just use OOR here without the full name because it's already defined previously.

Reply:
true, I'll revert the changes for this section.

Reply:
done